pypeclub/OpenPype
# General: OpenPype version updates

## Brief description
Version validation does not work as expected when newer version is available in local appdata.

## Changes
- info widget will show also build version
- validation of version is triggered immediately on tray launch
- added function to be able know if current version is higher version than studio expected
    - this can happen when a version in appdata is higher than latest version in version repository
- version is explicitly set to expected on restart from the dialog

## Testing notes:
1. Add zip to your version repository folder
2. Add zip with higher version into pypeclub folder in appdata
3. On launch the version from appdata should be used but dialog that newer version is used should popup

### Screenshot
When current version is older the dialog will display different labels.
![image](https://user-images.githubusercontent.com/43494761/150522109-6cdb597b-268e-4c60-8d2b-d13b3e75e33d.png)

## TODO
Add versions documentation.
